### DISCUSSION TITLE
Different Windows support

### DISCUSSION THREAD

**Discussion by vmarh** [Xpra-org/xpra#3751] [Q&A] [answered]
We use Xpra on Windows machines (virtualized) as a client (as opposed to a server) to display remote Linux windows. We encountered some issues with this approach, which lead us to open a discussion here and clear things up.

On some of those virtualized machines, we use Windows Server. On this setup, we have an issue where the remote Windows are rendered white - labels, etc however show up correctly. We found out that the issue with white windows was caused on some machines by VMWare graphical adapter and after changing it, the windows showed up correctly. This, however, worked only on some machines.

This therefore brought us to 2 questions:
We checked supported OS and only Windows 10 and WIndows 11 are listed and we wanted to confirm wherever you also fully support Windows Servers and which versions/editions (e.g. Windows Server 2019 Standard). Was this tested/confirmed as working?

Do you have experience with running Xpra as a client on virtualized Windows? Are there any incompatibilities ?

**Comment by totaam** [comment, marked as answer]
I do not have the resource to support Windows Server OS, so YMMV. That said, it shouldn't be too hard to fix.

The graphical adapter problem is usually a symptom of OpenGL issues, you may be able to workaround it by turning off OpenGL support if you can tolerate the loss of rendering performance.
https://github.com/Xpra-org/xpra/blob/master/docs/Usage/Client-OpenGL.md

It would be much better to create a ticket and find a way to either fix the rendering issue, or detect the broken configuration automatically so that xpra can switch to software rendering by itself.

> Do you have experience with running Xpra as a client on virtualized Windows?

Yes, every day.

> Are there any incompatibilities ?

Yes: OpenGL rendering, which is disabled with VirtualBox and may need to be disabled with other virtualized drivers.
